File: julia/guess.jl
#!/usr/bin/env julia
# File: guess.jl
# Name: D.Saravanan
# Date: 30/08/2020

""" Program for number guessing game """

option = "yes"

while option == "yes"

    N = rand(1:1:100)

    print("\nI 'm thinking of a number! Try to guess the number I 'm thinking of: ")
    num = parse(Int, readline())

    while num != N

        if num < N
            print("Too low! Guess again: ")
            num = parse(Int, readline())
        elseif num > N
            print("Too high! Guess again: ")
            num = parse(Int, readline())
        end
    end

    if num == N
        print("That's it! Would you like to play again? (yes/no) ")
        global option = readline()
    end
end

println("Thanks for playing!")

File: python/guess.py
#!/usr/bin/env python3
# File: guess.py
# Name: D.Saravanan
# Date: 16/08/2020
# Script for number guessing game

import numpy as np

option = 'yes'

while option == 'yes':

    N = np.random.randint(1, 100, size=1)

    num = int(input("\nI 'm thinking of a number! Try to guess the number I 'm thinking of: "))

    while num != N:

        if num < N:
            num = int(input("Too low! Guess again: "))
        elif num > N:
            num = int(input("Too high! Guess again: "))

    if num == N:
        option = input("That's it! Would you like to play again? (yes/no) ")

print("Thanks for playing!")
